tensorflow_quantum/datasets/spin_system_test.py

```diff
@@ -217,6 +217,108 @@ def test_param_resolver(self):
                                     rtol=1e-3)
 
 
+class TFIRectangularTest(tf.test.TestCase):
+    """Testing tfi_rectangular."""
+    # pylint: disable=C0103
```

Comment:
What exactly is this disabling ?

Reply:
Overloading the `setUpClass` method is making pylint sad because the method name is not lowercase. C0103 allows me to pass the pylint checks.